.github/labeler.yml

```diff
@@ -1,16 +1,40 @@
 # SPDX-License-Identifier: Apache-2.0
 
+CI/CD:
+  - changed-file:
+      - any-glob-to-any-file:
+          - .github/labeler.yml
+          - .github/workflows/**/*
+          - scripts/ruff.sh
+          - .pre-commit-config.yaml
+          - .pylintrc
+          - .spellcheck-en-custom.txt
+          - .spellcheck.yml
+          - tox.ini
+
+container:
+  - changed-file:
+      - any-glob-to-any-file:
+          - containers/**/*
+          - .dockerignore
+
 documentation:
   - changed-files:
       - any-glob-to-any-file:
           - CODE_OF_CONDUCT.md
+          - CONTRIBUTOR_RULES.md
+          - governance.md
+          - MAINTAINERS.md
           - README.md
+          - SECURITY.md
           - TROUBLESHOOTING.md
           - CONTRIBUTING/**/*
           - docs/**/*
+          - MAINTAINERS/**/*
```

Comment:
It looks like this supports globbing, so `*.md` may be better for the top level?

Reply:
Thought about that, but some Markdown files aren't docs necessarily, e.g. `PULL_REQUEST_TEMPLATE.md`